rest-api/edit-conversation-image-title/server/server.js

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+app.use(cors())
+app.use(bodyParser.json())
+
+async function editImageOrTitle(conversationId, imageURL, conversationTitle) {  
+  let data = {};
+
+  if (conversationTitle === undefined) {
+    data.photoUrl = imageURL;
+  } else if (imageURL === undefined) {
+    data.subject = conversationTitle;
+  } else {
+    data.subject = conversationTitle;
+    data.photoUrl = imageURL;
+  }
```

Comment:
```suggestion
  if (conversationTitle) {
    data.subject = conversationTitle;
  }
  
  if (imageUrl) {
    data.photoUrl = imageUrl;
  }
```

is quite a lot simpler